### DISCUSSION TITLE
Custom `highlights.scm` overriding default.

### DISCUSSION THREAD

**Discussion by LordMZTE** [nvim-treesitter/nvim-treesitter#3522] [Q&A] [answered]
I like to add "ligatures" to my code by setting conceals to certain keywords, for example [this file from my dotfiles](https://mzte.de/git/LordMZTE/dotfiles/src/branch/master/.config/nvim/queries/zig/highlights.scm).

This used to work just fine, but in some recent commit, nvim-treesitter must've changed the behaviour of these files so they override other `highlight.scm` files. The result of this is me only getting conceals and no syntax highlighting.

How can I add my own queries without overriding the default ones?

**Comment by clason** [comment, marked as answer]
This was changed in Neovim; you know need to explicitly mark queries as `; extends `

**Reply by LordMZTE** [reply]
Thank you!
